pkg/pillar/cmd/zedagent/handleconfig.go

```diff
@@ -185,12 +186,15 @@ func (s configSource) String() string {
 type configProcessingRetval int
 
 const (
-	configOK        configProcessingRetval = iota
-	configReqFailed                        // failed to request latest config
-	obsoleteConfig                         // newer config is already applied
-	invalidConfig                          // config is not valid (cannot be parsed, UUID mismatch, bad signature, etc.)
-	skipConfig                             // reboot or shutdown flag is set
-	defferConfig                           // not ready to process config yet
+	configOK             configProcessingRetval = iota
+	configReqFailed                             // failed to request latest config
+	invalidContentType                          // invalid proto content-type
+	invalidAuthContainer                        // config is not signed by the controller
+	obsoleteConfig                              // newer config is already applied
+	malformedConfig                             // unmarshaling failed
+	invalidConfig                               // config is not valid (cannot be parsed, UUID mismatch, bad signature, etc.)
```

Comment:
signature failure is listed for two of the values. Which one is used?

Reply:
Commit is removed.